modules/acs-architecture-overview.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * architecture/index.adoc
+
+:_module-type: REFERENCE
+[id="acs-architecture_{context}"]
+= {product-title} architecture
+
+{product-title} installs as a pod in your OpenShift or a Kubernetes cluster and it includes the following components:
+
+|===
+| Component | Description | Quantity
+
+| <<central,*Central*>>
+| Gathers and displays information from other components.
+| *1* for multiple clusters.
+
+| <<sensor,*Sensor*>>
+| Collects and augments data from the Collector.
+| *1* for each cluster.
+
+| <<scanner,*Scanner*>>
+| Scans images for vulnerabilities.
+| *1* for multiple clusters.
+
+| <<collector,*Collector*>>
+| Collects and monitors container activities.
+| *1* on each node.
+
+| <<admission-controller,*Admission controller*>> (optional)
+| Interacts with Kubernetes API server and prevents creating workloads that do not adhere to security policies.
+| *1* for each cluster.
+|===
+
+
+.{product-title} components
+image::acs-architecture.svg[{product-title} components]
```

Comment:
Looks like this isn't a Red Hat-branded diagram, so this (and all stackrox diagrams that you want to bring in) will probably need to get rebranded by our graphic designers.

@vikram-redhat Can you confirm if these diagrams will need to be Red Hat-branded by Jess? I thought you mentioned that that might already be in progress.

Reply:
Thanks @bergerhoffer @vikram-redhat 

I've marked it as a TODO for now and we can update the diagram once we have it finalized from the Graphics team.